src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -2461,6 +2482,54 @@ private static bool GetPartsForBitManipulation(ref BigInteger x, ref Span<uint>
             return x._sign < 0;
         }
 
+        private static bool IsEmpty([NotNullWhen(false)] uint[]? bits)
+        {
+            bool returnValue;
+
+            if (bits == null)
+            {
+                returnValue = true;
+            }
+            else
+            {
+                returnValue = false;
+            }
+
+            return returnValue;
+        }
+
+        private static bool TryGetInfinity(int length,
+            int sign,
+            [NotNullWhen(true)] out double? infinityValue)
+        {
+            bool returnValue;
+
+            // The maximum exponent for doubles is 1023, which corresponds to a uint bit length of 32.
```

Comment:
nit: The comment of `bit length of 32` isn't great (I know it was already there before this PR) as reading it sounds like `# of bits is more than 32` (implying a `32-bit integer`).

Reply:
The last sentence also looks to be no longer true.

Reply:
@tannergooding , I think, comment is evil and Uncle Bob ( http://cleancoder.com/products ) absolutely agree with me. I had wanted to remove this comment, but it can very hard to understand what's the magic value 1024 in line below. I'll try to change this comment to more clear.